ert_data/loader.py

```diff
@@ -82,33 +82,48 @@ def load_block_data(facade, observation_key, case_name):
     return data
 
 
-def load_summary_data(facade, observation_key, case_name):
-    data_key = facade.get_data_key_for_obs_key(observation_key)
-    return pd.concat(
-        [
-            _add_summary_observations(facade, data_key, case_name),
-            _add_summary_data(facade, data_key, case_name),
-        ]
-    )
-
-
 def _get_block_measured(ensamble_size, block_data):
     data = pd.DataFrame()
     for ensamble_nr in range(ensamble_size):
         data = data.append(pd.DataFrame([block_data[ensamble_nr]], index=[ensamble_nr]))
     return data
 
 
-def _add_summary_data(facade, data_key, case_name):
+def load_summary_data(facade, observation_key, case_name):
+    data_key = facade.get_data_key_for_obs_key(observation_key)
+    args = (facade, observation_key, data_key, case_name)
+    return pd.concat([
+        _get_summary_data(*args),
+        _get_summary_observations(*args).pipe(_remove_inactive_report_steps, *args)
```

Comment:
Think I would prefer calling pip in `_get_summary_observations`, but can be persuaded to have it here.

Reply:
Yeah, I can go either way. I would really have it as a processing pipeline, but I don't know enough pandas.

My argument is that the piping would go away if only `_get_summary_observations` would return the correct datas. Then you'd merely remove the pipe, as opposed to using tweezers on   `_get_summary_observations`.